config/config.go

```diff
@@ -958,7 +958,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		TimeoutPrevoteDelta:         500 * time.Millisecond,
 		TimeoutPrecommit:            1000 * time.Millisecond,
 		TimeoutPrecommitDelta:       500 * time.Millisecond,
-		TimeoutCommit:               1000 * time.Millisecond,
+		TargetRoundDuration:         3500 * time.Millisecond,
```

Comment:
[question] is there a guaranteed # of rounds per block? I assume no, so does this change mean each round will occur approx. every 3.5 seconds and correspondingly a block will occur approx. every 3.5 seconds * number of rounds in a block?

Reply:
[question] on a related note, do we have data on the # of rounds per block for our current testnets? Is that how the 3.5 seconds was arrived at? Potentially 15 second target block time ~= 4 rounds * 3.5 seconds (?)